sdks/python/setup.py

```diff
@@ -226,12 +307,6 @@ def run(self):
     ]),
     install_requires=REQUIRED_PACKAGES,
     python_requires=python_requires,
-    test_suite='nose.collector',
```

Comment:
What's alternative collector it will use? Can we ensure the collection result unchanged?

Reply:
I believe it will still use nose, just not via setup.py. We can compare nosetests.xml files to verify.

Reply:
Correct, `test_requires` and `test_suite` are for the `python setup.py test` and derivative workflows.  That whole stack is now deprecated in favor of extras and pep517.